services/src/main/resources/locale/addon/Gamification_en.properties

```diff
@@ -31,6 +31,7 @@ exoplatform.gamification.gamificationinformation.Date=Date
 exoplatform.gamification.gamificationinformation.domain.edit.label.description=Enter description
 exoplatform.gamification.gamificationinformation.domain.edit.label.title=Enter domain's title
 exoplatform.gamification.gamificationinformation.Domain=Domain
+exoplatform.gamification.gamificationinformation.domain.Autres=Autres
```

Comment:
Should be translated in english ?
```suggestion
exoplatform.gamification.gamificationinformation.domain.Autres=Others
```

Reply:
in addition, the key should also be in english

Reply:
Fixed, thanks